spec/requests/historical_banner_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+RSpec.describe "Documents" do
+  it "shows the history mode banner when the edition is in history mode" do
+    @edition = create(:edition, :published, :political, :past_government)
+    get document_path(@edition.document)
+    expect(response.body).to include(I18n.t!("documents.show.historical.title", document_type: @edition.document_type.label.downcase))
+    expect(response.body).to include(I18n.t!("documents.show.historical.description", government_name: @edition.government.name))
+  end
+
+  it "won't show the history mode banner when the edition was created under the current government" do
+    @edition = create(:edition, :published, :political, :current_government)
+    get document_path(@edition.document)
+    expect(response.body).not_to include(I18n.t!("documents.show.historical.title", document_type: @edition.document_type.label.downcase))
+    expect(response.body).not_to include(I18n.t!("documents.show.historical.description", government_name: @edition.government.name))
+  end
+
+  it "won't show the history mode banner when the edition is not political" do
+    @edition = create(:edition, :published, :not_political)
+    get document_path(@edition.document)
+    expect(response.body).not_to include(I18n.t!("documents.show.historical.title", document_type: @edition.document_type.label.downcase))
```

Comment:
Just for consistency, this test is not checking `title` _and_ `description` like the "created under the current government" test above. I suggest if it's needed for one, it's needed for the other - and if it's not needed for one, it's not needed for the other. I'd be tempted to move these assertions into a method we can call from both tests.

Reply:
So that's done because the description relies on there being a government attached. This test doesn't have a government because it's not political. I think that we should be testing it for at least one test but we physically can't for this one. Mind if I leave this as is?

Reply:
Sure 👍 thanks for explaining!